public/static/locales/en/admin-vault.json

```diff
@@ -2,5 +2,9 @@
   "title": "Forms Vault",
   "subTitle": "Your form responses",
   "formInput": "Form to get responses for",
-  "submitButton": "Look up responses"
+  "submitButton": "Look up responses",
+  "noResponse": "No Responses for selected form",
+  "nextButton": "Next",
+  "backButton": "Previous",
+  "removeButton": "Processed"
```

Comment:
I think this is probably something to iterate on, as "Processed" doesn't imply deletion, but since we don't have any users for this feature yet it's probably fine to keep for now. Just something to flag, maybe for design to help with later.

Reply:
Agreed 💯 
Never ever let me be the last word on 'copy'